server/lib/twitch.js

```diff
@@ -0,0 +1,64 @@
+const fetch = require('node-fetch');
+const WebSocket = require('ws');
+
+//ANDREW PLEASE
+function openSocketConnectionWithChannelId(channelId, oauth_token, callBack) {
```

Comment:
So, this has been irking me for a while, and I know I'm guilty of it. 
Can we standardize whether we are doing camel-case or snake-case for our variable names? I don't want both. One or the other.

Reply:
If it's easier, we can have a separate PR after this to address that issue in particular. Not a big problem